pkg/apis/resource/types.go

```diff
@@ -322,11 +341,11 @@ type Device struct {
 // DeviceCounterConsumption defines a set of counters that
 // a device will consume from a CounterSet.
 type DeviceCounterConsumption struct {
-	// SharedCounter defines the shared counter from which the
+	// CounterSet defines the set from which the
```

Comment:
Nit:
"CounterSet is the name of the set from which..."

Below:

"Counters defines the counters that will be consumed by the device."

Reply:
Fixed